modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -232,7 +232,7 @@ private static Map<String, Object> _getVariables(
 					return dateFormat.format(new Date());
 				}
 			).put(
-				"currentUserId", payloadJSONObject.getString("userId")
```

Comment:
@marcelabc @icgsbr this code was introduced to fix https://liferay.atlassian.net/browse/LPS-185225. 

Is it ok to change this?

Reply:
Thank you!
I added a fix so we don't make the same regression.